modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,139 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|The simple form of the node-selection constraint.
+|type: object
+|
+
+|`replicas`
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations`
+|Specifies component-specific pod tolerations.
+|type: array
 |
 |===
 
 
-.Configuration parameters passed to Query
+.Configuration parameters for the Query Frontend
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    options: {}
-|Configuration options that define the Query service.
+|`component`
+|Configuration of the Query Frontend component.
+|type: object
+|
+
+|`nodeSelector` (inside component)
+|The simple form of the node selection constraint.
+|type: object
+|
+
+|`replicas` (inside component)
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations` (inside component)
+|Specifies component-specific pod tolerations.
+|type: array
+|
+
+|`jaegerQuery`
+|Specifies the options specific to Jaeger Query.
+|type: object
+|
+
+|`enabled` (inside jaegerQuery)
+|When `enabled`, creates the Jaeger Query component,`jaegerQuery`.
+|type: boolean
+|
+
+|`ingress` (inside jaegerQuery)
+|Specifies the options for the Jaeger Query Ingress.
+|type: object
+|
+
+|`annotations` (inside jaegerQuery.ingress)
+|Specifies the annotations of the Ingress object.
+|type: object
+|
+
+|`host` (inside jaegerQuery.ingress)
+|Specifies the hostname of the Ingress object.
+|type: string
+|
+
+|`ingressClassName` (inside jaegerQuery.ingress)
+|The name of an IngressClass cluster resource. Defines which Ingress controller serves this Ingress resource.
+|type: string
 |
+
+|`route` (inside jaegerQuery.ingress)
+|Specifies OpenShift Route specific options.
+|type: object
 |
 
-|options:
-  log-level:
-|Logging level for Query.
-|`debug`, `info`, `warn`, `error`, `fatal`, `panic`
+|`termination` (inside jaegerQuery.ingress.route)
+|The termination type. The default is `edge`.
+|type: string (enum: insecure, edge, passthrough, reencrypt)
 |
 
-|options:
-  query:
-    base-path:
-|You can set the base path for all tempo-query HTTP routes to a non-root value: for example, `/tempo` will cause all UI URLs to start with `/tempo`. This can be useful when running `tempo-query` behind a reverse proxy.
-|`/<path>`
+|`type` (inside jaegerQuery.ingress)
+|Specifies the type of Ingress for the Jaeger Query UI. The following types are supported: `ingress`, `route`, and `none`.
```

Comment:
Is the capitalization of "Ingress" warranted here?

Reply:
Kubernetes vs OpenShift capitalization.
Thank you. I'm fixing it.